test/integration/driver-virtualbox.bats

```diff
@@ -270,7 +270,39 @@ findMemorySize() {
   [ "$status" -eq 0  ]
 }
 
+@test "$DRIVER: can create custom machine using disk size and memory size via env vars" {
+  export VIRTUALBOX_DISK_SIZE=$CUSTOM_DISKSIZE
+  export VIRTUALBOX_MEMORY_SIZE=$CUSTOM_MEMSIZE
+  run machine create -d $DRIVER $NAME
+  export VIRTUALBOX_DISK_SIZE=
+  export VIRTUALBOX_MEMORY_SIZE=
+  [ "$status" -eq 0  ]
+}
+
+@test "$DRIVER: check machine's memory size was set correctly by env var" {
+  findMemorySize
+  [[ ${output} == "${CUSTOM_MEMSIZE}"  ]]
```

Comment:
Ah yeah, thanks @nathanleclaire, I'd missed that one!! :disappointed:   I was removing the braces (to be consistent with the existing tests) from the block of tests that @slashk had kindly provided above.